style.css

```diff
@@ -344,3 +344,167 @@ header {
   height: 0.5rem;
   align-self: center;
 }
+
+@media only screen and (min-width: 768px) {
+  .head-section {
+    flex-direction: column;
+    background-image: url("desktop_header_bg.png");
```

Comment:
- [ ] Kindly add a fixed size for your logo in the toolbar aswell, right now on some of the larger screen sizes, the logo just gets too big and takes up too much of the screen which does not look as good as it is on some of the smaller screen sizes.

Reply:
done